compiler/rustc_parse/src/parser/expr.rs

```diff
@@ -971,7 +966,12 @@ impl<'a> Parser<'a> {
         }
     }
 
-    fn parse_dot_or_call_expr_with_(&mut self, mut e: P<Expr>, lo: Span) -> PResult<'a, P<Expr>> {
+    fn parse_dot_or_call_expr_with_(
+        &mut self,
+        mut e: P<Expr>,
+        lo: Span,
+    ) -> PResult<'a, (P<Expr>, bool)> {
```

Comment:
maybe a (doc) comment describing what this bool means would be nice,,

Reply:
galaxy brain: `enum ParseDotOrCall { Changed(P<Expr>), Unchanged(P<Expr>) }` :sweat_smile: 

but a bool is fine.

Reply:
```rust
enum ParseDotOrCall {
  Changed(P<Expr>),
  Unchanged(P<Expr>),
  CheckTheAddressJustToBeSure(P<Expr>),
}
```